tools/release_do_nothing.py

```diff
@@ -10,879 +10,859 @@
+        )
+        self.wait_for_done(message)
 
-            date_string = datetime.today().strftime("%Y%m%d")
+        if self.is_release_candidate:
             message = (
-                f"Archive the {rc_branch} branch by appending _{date_string} "
-                "to its name.\n"
-                f"e.g. rc_{date_string}\n\n"
-                f"({__file__} includes an explanation of this in the comments)."
+                f"Confirm that {self.strings.release} is marked as a "
+                f"pre-release on this page:\n"
+                "https://pypi.org/project/scitools-iris/#history"
             )
-            _wait_for_done(message)
-
+        else:
             message = (
-                "Follow the latest conda-forge guidance for creating a new "
-                "release candidate branch from the `main` branch:\n"
-                "https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds\n\n"
-                "Config file(s) should point to the `rc_iris` label.\n"
+                f"Confirm that {self.strings.release} is the tag shown on the "
+                "scitools-iris PyPI homepage:\n"
+                "https://pypi.org/project/scitools-iris/"
             )
-            rc_branch = _get_input(message, "Input the name of your new branch")
-
-        upstream_branch = rc_branch
-    else:
-        upstream_branch = "main"
-
-    # TODO: automate
-    message = (
-        "Checkout a new branch for the conda-forge changes for this release:\n"
-        "git fetch upstream;\n"
-        f"git checkout upstream/{upstream_branch} -b {release_strings.tag};\n"
-    )
-    _wait_for_done(message)
-
-    message = (
-        "Update ./recipe/meta.yaml:\n\n"
-        f"- The version at the very top of the file: {release_strings.release}\n"
-        f"- The sha256 hash: {sha256}\n"
-        "- Requirements: align the packages and pins with those in the Iris repo\n"
-        "- Maintainers: update with any changes to the dev team\n"
-        "- MAKE SURE everything else is correct - plenty of other things "
-        "might need one-off changes.\n"
-    )
-    _wait_for_done(message)
-
-    # TODO: automate
-    message = (
-        "No other file normally needs changing in iris-feedstock, so push up "
-        "the changes to prepare for a Pull Request:\n"
-        f'git commit -am "Recipe updates for {release_strings.tag} .";\n'
-        f"git push -u origin {release_strings.tag};"
-    )
-    _wait_for_done(message)
+        self.wait_for_done(message)
 
-    message = (
-        f"Follow the Pull Request process to get {release_strings.tag} branch "
-        f"merged into upstream/{upstream_branch} .\n"
-        "Specific conda-forge guidance will be automatically given once the "
-        "PR is created."
-    )
-    _wait_for_done(message)
+        def validate(sha256_string: str) -> str:
+            try:
+                sha_len = int(sha256_string, 16).bit_length() + 1
```

Comment:
This doesn't actually work - the `bit_length()` of the string can be anywhere from `253` to `256`. The correct two things to check are:

- That the string can be validly parsed as a 16-bit number (`int(sha256_string, 16)`)
- That the string is 64 characters long

Reply:
bc7f6bca2